frontend/src/pages/devices/List.tsx

```diff
@@ -1,10 +1,11 @@
 import React from 'react'
-import { Badge, Button, OverlayTrigger, Tooltip } from 'react-bootstrap'
+import { Badge, Button, OverlayTrigger, Table, ListGroup, Tooltip } from 'react-bootstrap'
```

Comment:
Alphabetize (there's not a good reason for this, it's just the way I've been ordering them).